tools/README.md

```diff
@@ -1,7 +1,19 @@
 # Discovery Tools
 
+This is a Go interface implementation to resolve image reference name.
+
+## Build
+
+Developer can build up the binary tools via source code.
```

Comment:
This is hopefully true of all open-source projects ;).  I'd drop this line and replace the following one with:

> Build the `oci-discovery` executable from the project root with:

or similar.